src/executor/Deleter.py

```diff
@@ -0,0 +1,15 @@
+from typing import Union
+
+from ..data.MetaDataItem import MetaDataItem
+from ..signals import DeleteSignal
+from ..data.VideoItem import VideoItem
+from .iExecutor import iExecutor
+
+
+class Deleter(iExecutor):
```

Comment:
Seems like a pretty dangerous class... XD
How are we gonna use it?

Reply:
Honestly, it probably isn't needed given I have a safer version of it with the DatabaseCleaner, but it could be useful later